pages/cartPage.js

```diff
@@ -0,0 +1,36 @@
+import { expect } from '@playwright/test';
+import { formatProductName } from '../utils/stringHelpers';
+
+export class CartPage {
+  constructor(page) {
+    this.page = page;
+    this.itemRow = this.page.locator('[data-test="inventory-item"]');
+    this.checkoutButton = this.page.locator('[data-test="checkout"]');
+  }
+
+  async verifyProductInCart(productName, notDisplayed = false, quantity = '1') {
+    //Verify existence of a given product in cart, by default checking quantity=1 and product visibility=true
+    const filteredLocator = this.itemRow
+      .filter({ hasText: productName })
+      .filter({ has: this.page.locator('[data-test="item-quantity"]', { hasText: quantity }) })
+    if (notDisplayed) {
```

Comment:
I would change it to [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator).